metrics/metrics.go

```diff
@@ -0,0 +1,165 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/rcrowley/go-metrics"
+	"net/http"
+	"time"
+)
+
+type skipperMetrics map[string]interface{}
+
+// Options for initializing metrics collection.
+type Options struct {
+	// Network address where the current metrics values
+	// can be pulled from. If not set, the collection of
+	// the metrics is disabled.
+	Listener string
+
+	// Common prefix for the keys of the different
+	// collected metrics.
+	Prefix string
+
+	// If set, garbage collector metrics are collected
+	// in addition to the http traffic metrics.
+	EnableDebugGcMetrics bool
+
+	// If set, Go runtime metrics are collected in
+	// addition to the http traffic metrics.
+	EnableRuntimeMetrics bool
+}
+
+const (
+	KeyRouteLookup     = "routelookup"
+	KeyFilterRequest   = "filter.%s.request"
+	KeyFiltersRequest  = "filters.request.%s"
+	KeyProxyBackend    = "backend.%s"
+	KeyFilterResponse  = "filter.%s.response"
+	KeyFiltersResponse = "filters.response.%s"
+	KeyResponse        = "response.%d.%s.skipper.%s"
+)
+
+var reg metrics.Registry
+
+// Initializes the collection of metrics.
+func Init(o Options) {
+	if o.Listener == "" {
+		log.Infoln("Metrics are disabled")
+		return
+	}
+
+	r := metrics.NewRegistry()
+	if o.EnableDebugGcMetrics {
+		metrics.RegisterDebugGCStats(r)
+		go metrics.CaptureDebugGCStats(r, 5e9)
```

Comment:
Any way to make that `5e9` more readable? Like `(5 * Time.second)` or something like that?